packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts

```diff
@@ -0,0 +1,52 @@
+// @group:suite
+// @retry=2
+
+describe('Check Vertcoin XPUB', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic: 'all all all all all all all all all all all all',
```

Comment:
performacne nitpick: might be better to use another seed, where discovery is faster

Reply:
we will consider it for next versions of tests
1) seed with less transactions
2) checking xpub value
3) checking all xpubs for all coins in one test to optimize test performance
now are in process of learning :)